### PR TITLE
VPN Ban Notice

### PR DESCRIPTION
**Pull request by murpii**
Steps to follow when a VPN leads to a ban.

### CODE REVIEW

File: www/vpn-ban-notice/index.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+              <li><a href="/binds/">Useful&nbsp;Binds</a></li>
+              <li><a href="/auto-font/">Auto-Font</a></li>
+              <li><a href="/settingscommands/">Settings&nbsp;&amp;&nbsp;Commands</a></li>
+              <li><a href="/renames/">Renames</a></li>
+              <li><a href="/funding/">Funding</a></li>
+            </ul>
+          </menu>
+        </div>
+      </header>
+      <section>
```
Comment (def-) [inline www/vpn-ban-notice/index.html:71]: This should all be replaced by
```suggestion
---
layout: default
title: VPN Ban Notice - DDraceNetwork
menu: |
  <ul>
    <li><a href="/#news">News</a> (<a href="/news/">all</a>)</li>
    <li><a href="/server/">Server&nbsp;Features</a></li>
    <li><a href="/client/">Client&nbsp;Features</a></li>
    <li><a href="/map/">Map&nbsp;Features</a></li>
    <li><a href="/howto/">How&nbsp;to&nbsp;Map</a></li>
    <li><a href="/rules/">Mapping&nbsp;Rules</a> / <a href="/guidelines/">Guidelines</a></li>
    <li><a href="/binds/">Useful&nbsp;Binds</a></li>
    <li><a href="/auto-font/">Auto-Font</a></li>
    <li><a href="/settingscommands/">Settings&nbsp;&amp;&nbsp;Commands</a></li>
    <li><a href="/renames/">Renames</a></li>
    <li><a href="/funding/">Funding</a></li>
  </ul>
---
```

File: www/vpn-ban-notice/index.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            <li>Using a VPN intentionally to mask your IP address.</li>
+			<li>A False-Positive.</li>
+          </ul>
+          <p>Please check your settings and disable any active VPN services.</p>
+          <p>If you are not using a VPN and believe this ban is a mistake, please join our <a href="/discord">Discord</a> and create a ban-appeal ticket.</p>
+        </div>
+      </section>
+    </article>
+  </body>
+</html>
```
Comment (def-) [inline www/vpn-ban-notice/index.html:87]:
```suggestion
```

**Review by def- — CHANGES_REQUESTED** [review]

**Comment by murpii** [comment]
I just used /renames/html.index as template :D

**Review by def- — APPROVED** [review]

**Comment by def-** [comment]
renames/index.html looks fine though.

**Comment by murpii** [comment]
Oh yeah, I just took it straight from ddnet.org's server :D

**Pull request merged**